services/dao-service.ts

```diff
@@ -49,6 +49,7 @@ export default class DAO {
       await network.loadContract();
 
       if (!skipAssignment) this._network = network;
+      else this._networkAux = network;
```

Comment:
Like we spoke, it's easier to have the component itself initialize a `new DAO()` and then `loadNetwork('0xCustomNetworkAddress')` than having an auxiliary network _that might_ get confused on the if/else and return the wrong network on the `getNetwork()` getter.

We also ensure that all the methods are called on the network that we want to edit without adding any more complexity :)